world/login_server.cpp

```diff
@@ -366,15 +366,15 @@ bool LoginServer::Connect()
 				}
 				else {
 					LogInfo(
-						"Could not connect to Legacy Loginserver: [{0}:{1}]",
-						LoginServerAddress,
-						LoginServerPort
+					"Could not connect to Legacy Loginserver: [{0}:{1}]",
+					m_loginserver_address,
+					m_loginserver_port
```

Comment:
Unintentional loss of indent here I think

Reply:
Fixed